src/components/presentation/uiHelpers.ts

```diff
@@ -113,7 +112,7 @@ export const calculateLoraSignalStrength = (signalBars: SignalStrengths) => {
     "0": LoraOff,
     "1": LoraOne,
     "2": LoraTwo,
-    "3": LoraThree,
+    "3": LoraFull,
```

Comment:
interesting we're considering 3 and 4 both 'full' now.

Reply:
There were only 4 images provided for this simplified version of the LoRa icon so it was easier to use the "full" icon for both 3 and 4 instead of breaking my TypeScript enum by having only 4 levels for LoRa as opposed to 5 levels for Wi-Fi and cellular, which share the same enum definition.